src/components/Global/BaseSnackbar.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <v-snackbar
+    v-model="showSnackBar"
+    :color="color"
+    class="base-snackbar"
+    position="absolute"
+    location="bottom"
+    data-cy="auth-snackbar"
+    v-bind="$attrs"
+  >
+    <slot />
+    <slot name="actions">
+      <v-btn data-cy="close-snackbar" icon variant="text" @click="clear">
+        <v-icon icon="mdi-close" />
+      </v-btn>
+    </slot>
```

Comment:
I think this will fix the alignment bug
```suggestion
    <template #actions>
      <slot name="actions">
        <v-btn data-cy="close-snackbar" icon variant="text" @click="clear">
          <v-icon icon="mdi-close" />
        </v-btn>
      </slot>
    </template>
```

Reply:
Actually simpler yet. There's no need to customize the close button, really. So until we hit a use-case where we need to do that, all we have to change is swapping the slot for a template

Reply:
Actually simpler yet. There's no need to customize the close button, really. So until we hit a use-case where we need to do that, all we have to change is swapping the slot for a template
```suggestion
    <template #actions>
      <v-btn data-cy="close-snackbar" icon variant="text" @click="clear">
        <v-icon icon="mdi-close" />
      </v-btn>
    </template>
```